generatebundlefile/scripts/generate_bundle.sh

```diff
@@ -27,23 +27,24 @@ BASE_DIRECTORY=$(git rev-parse --show-toplevel)
 chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile 
 
 # Faster way to install Cosign compared to go install github.com/sigstore/cosign/cmd/cosign@v1.5.1
-curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
+COSIGN_URL=$(curl -s https://api.github.com/repos/sigstore/cosign/releases/latest \
 | grep 'browser_download_url.*cosign-linux-amd64"' \
 | cut -d '"' -f 4 \
-| tr -d \" \
-| wget -qi -
+| tr -d \")
 
+curl -OL $COSIGN_URL
```

Comment:
Here as well, you could curl directly into tar and save yourself some temporary directories (and their cleanup).